docker-compose.yml

```diff
@@ -90,6 +90,13 @@ services:
     stdin_open: true
     tty: true
 
+  redis:
```

Comment:
There are conflicts too complicated to resolve on GitHub, you want me to do it or you want to?